lib/htmlValidator.js

```diff
@@ -55,7 +55,7 @@ module.exports = function(app) {
       // get the html for this specific render
       app.render(view, model, function(err, html) {
 
-        if (process.env.NODE_ENV === 'development' && !validatorDisabled) {
+        if (process.env.NODE_ENV === 'development' && !validatorDisabled && req.header !== 'Partial' && !model._disableValidator) {
```

Comment:
- `req.header` seems wrong 
- these escape hatches need documentation in the README